lib/util.js

```diff
@@ -233,6 +248,9 @@ exports.getVideoID = function(link) {
   if (!id) {
     throw new Error('No video id found: ' + link);
   }
+  if(id === 'playlist') {
+    throw new Error('Playlists not supported (yet)');
+  }
```

Comment:
maybe thatll come one day... should fix the error descriped in https://github.com/fent/node-ytdl-core/issues/102